xbrowse_server/staticfiles/js/base/select_variants.js

```diff
@@ -239,20 +264,21 @@ window.SelectVariantsView = Backbone.View.extend({
 
         this.ref_freq_sliders = {};
         _.each(this.reference_populations, function(pop) {
-            var freqSliderInitialVal = 9;
+            var freqSliderInitialVal = 11;
             var newslider = that.$('.freq-slider[data-population="' + pop.slug + '"]').slider({
                 min: 1,
-                max: 9,
+                max: 11,
```

Comment:
It might be better to use `freqSliderInitialVal` here instead of repeating the value

Reply:
good point